### PR TITLE
Fix code sample in troubleshooting guide

### LINKED ISSUE #36156: Monitor Ingestion Samples Issue
1.
**Section** [link](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/monitor/azure-monitor-ingestion/src/samples/java/com/azure/monitor/ingestion/ReadmeSamples.java#L122):
image
**Reason**:
java.lang.IllegalStateException: endpoint is required to build the client.

**Suggestion**:
	Update code to 
```
LogsIngestionClient logsIngestionClient = new LogsIngestionClientBuilder()
	            .endpoint("<data-collection-endpoint>")
	            .credential(credential)
	            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
	            .buildClient();
```
@sandeep-sen, @josefree, @scottaddie and @srnagar for notification.

### PR DESCRIPTION
**Pull request by srnagar**
Fixes #36156

### CODE REVIEW

**Review by jairmyree — APPROVED** [review]

File: sdk/monitor/azure-monitor-ingestion/src/samples/java/com/azure/monitor/ingestion/ReadmeSamples.java [hunk 1]
```diff
@@ -120,6 +120,7 @@ public void tsgEnableHttpLogging() {
 
         // BEGIN: readme-sample-enablehttplogging
         LogsIngestionClient logsIngestionClient = new LogsIngestionClientBuilder()
+            .endpoint("<data-collection-endpoint")
```
Comment (scottaddie) [inline sdk/monitor/azure-monitor-ingestion/src/samples/java/com/azure/monitor/ingestion/ReadmeSamples.java:123]: Add missing > in placeholder

File: sdk/monitor/azure-monitor-ingestion/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ Reviewing the HTTP request sent or response received over the wire to/from the A
 
 ```java readme-sample-enablehttplogging
 LogsIngestionClient logsIngestionClient = new LogsIngestionClientBuilder()
+    .endpoint("<data-collection-endpoint")
```
Comment (scottaddie) [inline sdk/monitor/azure-monitor-ingestion/TROUBLESHOOTING.md:37]: Add missing > in placeholder

**Review by scottaddie — APPROVED** [review]
A couple small nits, and then it looks good to me

File: sdk/monitor/azure-monitor-ingestion/TROUBLESHOOTING.md (hunk 2 above)
Comment (joshfree) [reply sdk/monitor/azure-monitor-ingestion/TROUBLESHOOTING.md:37]: nice catch

**Review by joshfree — APPROVED** [review]

**Pull request merged**